cfme/fixtures/service_fixtures.py

```diff
@@ -158,3 +158,27 @@ def order_service(appliance, provider, provisioning, dialog, catalog, request):
     vm_name = '{}0001'.format(catalog_item.prov_data['catalog']['vm_name'])
     vm = appliance.collections.infra_vms.instantiate(vm_name, provider)
     vm.cleanup_on_provider()
+
+
+@pytest.fixture(scope="module")
+def service_vm(appliance, provider, catalog_item):
```

Comment:
Most of the code looks like order_service fixture 
https://github.com/ManageIQ/integration_tests/blob/e76d390991c98d95ac740fb2116cd0f1e4d2ec09/cfme/fixtures/service_fixtures.py#L137

Can we tweak the same method to return service and vm name too ?

Reply:
@sshveta I already mention this possibility in `PR` explanation.

Actually, `order_service` is different than this fixture as its using `request` input for taking decision related to `catalog_item` and yielding `catalog_item`. 

I would like to use this fixture as `setup` for tests (custom button and some service test can use it), so I am `skipping test` if service provisioning `fails` and `order_service` asserting `success` mean it can be a part of test.

So, as per me better to put both separate fixtures and in future if FA owner like to combine he/she can work on it. Also If i add changes it can break service test which i don't want.

Reply:
Yes both fixtures are different in what they yield but with some testing can be merged . I think that's Niyaz's part and can be done later .